kong/plugins/request-transformer/schema.lua

```diff
@@ -10,8 +10,18 @@ local function check_for_value(value)
   return true
 end
 
+local function check_method(value)
+  if not value then return true end
+  local method = value:upper()
+  local ngx_method = ngx["HTTP_"..method]
+  if not ngx_method then
+    return false, method.." is not supported"
```

Comment:
By convention our codebase returns `nil` on error. Some parts are not yet updated, but new code should comply.